fr/development/configuration.rst

```diff
@@ -509,159 +509,20 @@ restaurer avec la même clé, et la configuration de cache comme elle était
-
-.. php:method:: read($key)
-
-    :param string $key: Le nom de la clé ou l'identifieur à charger.
-
-    Cette méthode devrait charger/parser les données de configuration
-    identifiées par ``$key`` et retourner un tableau de données dans le fichier.
-
-.. php:method:: dump($key)
-
-    :param string $key: L'identifieur dans lequel écrire.
-    :param array $data: La donnée à supprimer.
-
-    Cette méthode doit supprimer/stocker la donnée de configuration fournie à
-    une clé identifié par ``$key``.
-
-Moteurs de Configuration intégrés
-=================================
-
-.. php:namespace:: Cake\Core\Configure\Engine
-
-Fichiers de Configuration PHP
------------------------------
-
-.. php:class:: PhpConfig
-
-Vous permet de lire les fichiers de configuration de votre application qui sont
-stockés en fichiers PHP simples. Vous pouvez lire soit les fichiers à partir de
-votre config, soit des répertoires configs du plugin en utilisant la
-:term:`syntaxe de plugin`. Les fichiers **doivent** retourner un tableau.
-Un fichier de configuration d'exemple ressemblerait à cela::
-
-    return [
-        'debug' => 0,
-        'Security' => [
-            'salt' => 'its-secret'
-        ],
-        'App' => [
-            'namespace' => 'App'
-        ]
-    ];
-
-Chargez votre fichier de configuration personnalisé en insérant ce qui suit dans
-**config/bootstrap.php**::
-
-    Configure::load('customConfig');
-
-Fichiers de Configuration Ini
------------------------------
-
-.. php:class:: IniConfig
-
-Vous permet de lire les fichiers de configuration qui sont stockés en fichiers
-.ini simples. Les fichiers ini doivent être compatibles avec la fonction php
-``parse_ini_file()``, et bénéficie des améliorations suivantes:
-
-* Les valeurs séparées par des points sont étendues dans les tableaux.
-* Les valeurs de la famille des booléens comme 'on' et 'off' sont converties
-  en booléens.
-
-Un fichier ini d'exemple ressemblerait à cela::
-
-    debug = 0
-
-    [Security]
-    salt = its-secret
-
-    [App]
-    namespace = App
-
-Le fichier ini ci-dessus aboutirait aux mêmes données de configuration que dans
-l'exemple PHP du dessus. Les structures de tableau peuvent être créées soit à
-travers des valeurs séparées de point, soit des sections. Les sections peuvent
-contenir des clés séparées de point pour des imbrications plus profondes.
-
-Fichiers de Configuration Json
-------------------------------
-
-.. php:class:: JsonConfig
+Moteurs de Configuration
+------------------------
 
-Vous permet de lire / effacer les fichiers de configuration qui sont stockés en
-chaînes encodées JSON dans des fichiers .json.
+CakePHP vous permet de charger des configurations provenant de plusieurs sources
+et formats de données différents et vous donne accès à un système extensible pour
+`créer vos propres moteurs de configuration
+<https://api.cakephp.org/3x/class-Cake.Core.Configure.ConfigEngineInterface.html>`__.
```

Comment:
Could you fix this incorrect link, as it has been fixed in [#5182](https://github.com/cakephp/docs/pull/5182/files#diff-3ba80c6ca71316c8c02c0514f73abba6L491)? I will merge his pull request soon.